commit a6f61ce289d94669268ac53e51ca174db031b762 (refs/changes/70/1970/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-05-18T10:19:08-07:00 (2 years, 5 months ago)
    
    TOOLS-1778 sdcadm refactor: move steps/agent-services.js to steps/sapi.js
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/cli/do_add_new_agent_svcs.js b/lib/cli/do_add_new_agent_svcs.js
index e582a58..39502fc 100644
--- a/lib/cli/do_add_new_agent_svcs.js
+++ b/lib/cli/do_add_new_agent_svcs.js
@@ -19,7 +19,7 @@ function do_add_new_agent_svcs(subcmd, opts, args, cb) {
         'Warning: "sdcadm experimental add-new-agent-svcs" is deprecated.');
     console.error('Use "sdcadm experimental update-other" or');
     console.error('"sdcadm experimental update-agents".\n');
-    steps.agentServicesEnsureCreated({
+    steps.sapiEnsureAgentServices({
         progress: this.progress,
         sdcadm: this.sdcadm,
         log: this.log
diff --git a/lib/cli/do_update_other.js b/lib/cli/do_update_other.js
index cdeeae4..d8f55b7 100644
--- a/lib/cli/do_update_other.js
+++ b/lib/cli/do_update_other.js
@@ -828,7 +828,7 @@ function do_update_other(subcmd, opts, args, cb) {
             });
         },
 
-        steps.agentServicesEnsureCreated,
+        steps.sapiEnsureAgentServices,
 
         /*
          * Previously CloudAPI configuration did not include an external
diff --git a/lib/sdcadm.js b/lib/sdcadm.js
index 8cca91c..f601119 100644
--- a/lib/sdcadm.js
+++ b/lib/sdcadm.js
@@ -1668,7 +1668,7 @@ SdcAdm.prototype.updateAgents = function updateAgents(options, callback) {
             });
         },
 
-        steps.agentServicesEnsureCreated,
+        steps.sapiEnsureAgentServices,
 
         function downloadAgentsshar(ctx, next) {
             if (filepath) {
diff --git a/lib/steps/index.js b/lib/steps/index.js
index eb5f994..12ba0a0 100644
--- a/lib/steps/index.js
+++ b/lib/steps/index.js
@@ -24,7 +24,7 @@ module.exports = {};
 [
     'no-rabbit',
     'update_vm_size',
-    'agent-services'
+    'sapi'
 ].forEach(function (modName) {
     var mod = require('./' + modName);
     Object.keys(mod).forEach(function (symbol) {
diff --git a/lib/steps/agent-services.js b/lib/steps/sapi.js
similarity index 98%
rename from lib/steps/agent-services.js
rename to lib/steps/sapi.js
index 39914f1..cc0431d 100644
--- a/lib/steps/agent-services.js
+++ b/lib/steps/sapi.js
@@ -36,14 +36,14 @@ var DRY_RUN = false; // An off-switch for dev/testing.
  * item getting overwritten by a separate `sdcadm.history.saveHistory`
  * during this same command. It would be good to improve this at some point.
  */
-function agentServicesEnsureCreated(arg, cb) {
+function sapiEnsureAgentServices(arg, cb) {
     assert.object(arg, 'arg');
     assert.func(arg.progress, 'arg.progress');
     assert.object(arg.log, 'arg.log');
     assert.object(arg.sdcadm, 'arg.sdcadm');
     assert.func(cb, 'cb');
 
-    var log = arg.log.child({component: 'agentServicesEnsureCreated'}, true);
+    var log = arg.log.child({component: 'sapiEnsureAgentServices'}, true);
     var sdcadm = arg.sdcadm;
     var progress = arg.progress;
 
@@ -300,7 +300,7 @@ function agentServicesEnsureCreated(arg, cb) {
 //---- exports
 
 module.exports = {
-    agentServicesEnsureCreated: agentServicesEnsureCreated
+    sapiEnsureAgentServices: sapiEnsureAgentServices
 };
 
 // vim: set softtabstop=4 shiftwidth=4:
